src/components/cluster/detail/WorkerNodesAzure.js

```diff
@@ -21,21 +22,29 @@ export const LineDiv = styled.div`
 `;
 
 function WorkerNodesAzure({ instanceType, nodes, showScalingModal }) {
+  const instanceTypeText = instanceType
+    ? // prettier-ignore
+      `${instanceType.cpu_cores} CPUs, ${(instanceType.memoryInMb / 1000.0).toFixed(1)} GB RAM`
```

Comment:
@ograu: oops, for the CPU, this one needs to be `instanceType. numberOfCores`

Reply:
🤦‍♂ opening a pr...